.azure-pipelines-steps.yml

```diff
@@ -0,0 +1,50 @@
+# Build steps common to all platforms
+
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: $(node_version)
+  displayName: 'Install Node.js'
+
+- task: Npm@1
+  inputs:
+    command: custom
+    customCommand: i -g npm
+  displayName: 'Update npm'
+  condition: eq(variables['node_version'], '6.x')
+
+- task: Npm@1
+  inputs:
+    command: custom
+    customCommand: ci
+  displayName: 'Install dependencies'
+
+- bash: |
+    npm test -- -- --reporter mocha-junit-reporter --reporter-options mochaFile=./test-results.xml
+  displayName: 'Run tests'
+
+- task: PublishTestResults@2
+  inputs:
+    testResultsFiles: '**/test-results.xml'
+    testRunTitle: $(os_name) Node $(node_version)
+  displayName: 'Publishing the test results'
+  condition: always()
+
+- bash: |
+    export DISPLAY=$(DISPLAY)
```

Comment:
`DISPLAY` should be an environment variable, meaning you don't need this EXPORT statement, right? If for some reason it's not already an environment variable (I am assuming matrix variables become environment variables), I would set it via the "env" property on the bash task.

Reply:
I didn't know this, thanks.